timemachine/fe/free_energy.py

```diff
@@ -839,12 +838,16 @@ def get_equilibrated_initial_state(initial_state: InitialState) -> InitialState:
         assert len(xs) == len(boxes) == 1
         x0 = xs[0]
         box0 = boxes[0]
+        v0 = ctxt.get_v_t()
```

Comment:
```
x0 = xs[-1]
box0 = boxes[-1]
```

(and `assert x0 == ctxt.get_x_t()` to be in sync with v0)

Reply:
(edit: missed` store_x_interval==0`)

Reply:
Do we need the `assert x0 == ctxt.get_x_t()`? We have tests for this https://github.com/proteneer/timemachine/blob/master/tests/test_md.py#L70